storage_file/models/storage_backend.py

```diff
@@ -151,5 +151,8 @@ def _get_url_for_file(self, storage_file):
 
     def _register_hook(self):
         super()._register_hook()
-        self.search([]).action_recompute_base_url_for_files()
+        backends = self.search([])
```

Comment:
we could maybe filter this like:
```py
self.serch([]).filtered(lambda x: x._get_base_url_for_files() != x.base_url_for_files)
```
to avoid computing at all if not needed.

Reply:
Done